lib/cn/omf.h

```diff
@@ -167,6 +165,38 @@ static_assert(HSE_KVS_PFX_LEN_MAX <= UINT8_MAX,
 
 static_assert(HBLOCK_HDR_PAGES == 1, "Hblock header spanning more than 1 page has not been tested");
 
+
+/*****************************************************************
+ *
+ * Vgroup Map OMF
+ *
+ ****************************************************************/
+
+#define VGROUP_MAP_MAGIC   ((uint32_t)('v' << 24 | 'g' << 16 | 'p' << 8 | 'm'))
```

Comment:
Put the m before the p? Nice magic :).

Reply:
@beckerg suggested using something like this in an earlier PR of mine vs using 0xface0ff etc. The 'm' at the end indicates that this is a magic string.